blazor/bullet-chart/events.md

```diff
@@ -160,4 +160,48 @@ The [LegendRender](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Chart
         // Here, you can customize the code.
     }
 }
+
+```
+## PointerClick
+
+The [PointerClick](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.BulletChartEvents.html#Syncfusion_Blazor_Charts_BulletChartEvents_PointerClick) event triggers when pointer click.
```

Comment:
The [PointerClick](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.BulletChartEvents.html#Syncfusion_Blazor_Charts_BulletChartEvents_PointerClick) event triggers when the mouse or touch pointer is clicked on either the feature measure value or the target elements.

Use the above content.

Reply:
Content Changed